modules/quality/include/opencv2/quality/qualitymae.hpp

```diff
@@ -0,0 +1,75 @@
+#ifndef OPENCV_QUALITY_MAE_HPP
+#define OPENCV_QUALITY_MAE_HPP
+
+#include "qualitybase.hpp"
+
+
+namespace cv
+{
+
+namespace quality
+{
+
+/** @brief Flags to choose which algorithm MAE should use.
+ */
+enum MAEStatsFlags
+{
+    MAE_Max,
+    MAE_Mean
```

Comment:
Please use uppercase for enum values/constants: https://github.com/opencv/opencv/wiki/Coding_Style_Guide#naming-conventions